packages/block-library/src/paragraph/style.scss

```diff
@@ -28,6 +28,12 @@
 	font-style: normal;
 }
 
+// :root increases CSS specificity
+// This improves rendering consistency within the editor and site.
+:root p.has-line-height {
+	line-height: var(--wp--core-paragraph--line-height, inherit);
```

Comment:
is this in line with the global styles convention? Is this variable supported by the global styles theme.json, should we add a fallback to the block-less variable? cc @jorgefilipecosta